src/Data/Functor/Internal/Linear/Applicative.hs

```diff
@@ -0,0 +1,79 @@
+{-# OPTIONS_HADDOCK hide #-}
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Data.Functor.Internal.Linear.Applicative
+  (
+    Applicative(..)
+  ) where
+
+import Data.Functor.Internal.Linear.Functor
+import Prelude.Linear.Internal
+import qualified Control.Monad.Trans.Reader as NonLinear
+import Data.Monoid.Linear hiding (Sum)
+import Data.Functor.Compose
+import Data.Functor.Const
+import Data.Functor.Identity
+
+-- # Applicative Defintion
```

Comment:
```suggestion
-- # Applicative definition
```

(mostly for the missing 'i', but I think that the capital 'D' is unwarranted as well)